database/src/main/scala/cromwell/database/sql/tables/WorkflowStoreEntry.scala

```diff
@@ -0,0 +1,14 @@
+package cromwell.database.sql.tables
+
+import java.sql.Timestamp
+
+case class WorkflowStoreEntry
+(
+  workflowUuid: String,
+  workflowSource: String,
+  workflowInputs: Option[String],
+  workflowOptions: Option[String],
+  state: String, // TODONE: This is a database column, and **the column** has no restrictions on the string stored.
```

Comment:
In that case, I think I ought to replace the rather vague `initialize` method to the DB with a specific state update invocation and push knowledge of `Submitted`, `Running` and `Restartable` entirely up into the engine.